transfer-archive/app/main.py

```diff
@@ -32,7 +33,9 @@ def _log_startup_info(settings: Settings) -> None:
                 f"IAM_SECRET_ACCESS_KEY={settings.iam_secret_access_key}, "
                 f"S3_ENDPOINT_URL={settings.s3_endpoint_url}, "
                 f"S3_PATH={settings.s3_path}, "
-                f"SAS_GENERATOR_BASE_URL={settings.sas_generator_base_url}")
+                f"SAS_GENERATOR_BASE_URL={settings.sas_generator_base_url}, "
+                f"GENERATION_NAME={settings.generation_name}, "
+                f"OBJEKT_ID={settings.objekt_id}")
```

Comment:
```suggestion
                f"OBJEKT_ID={settings.objekt_id}"
            )
```

Endrer du indentering på disse?
Også
```
logger.info(f"Running transfer-file with the following environment variables: "
            f"AZURE_BASE_URL={settings.azure_base_url}, "
```
til
```
logger.info(
    f"Running transfer-file with the following environment variables: "
    f"AZURE_BASE_URL={settings.azure_base_url}, "
```